stylesheets/_component.modals.scss

```diff
@@ -107,8 +107,14 @@
 
 // Title text within header
 .modal__title {
-    margin: 0;
+    border: 0;
+    font-size: 16px;
     line-height: $modal-title-line-height;
+    margin: 0;
+
+    h1 {
+        border: 0;
```

Comment:
Curious as to why we needed the `border: 0`.

Reply:
It was for doc styles, I will move it there